frontend/src/components/widgets/Radio/Radio.tsx

```diff
@@ -80,13 +84,32 @@ class Radio extends React.PureComponent<Props, State> {
         <RadioGroup
           onChange={this.onChange}
           value={this.state.value.toString()}
-          disabled={disabled}
+          disabled={isDisabled}
         >
           {options.map((option: string, index: number) => (
             <UIRadio
               key={index}
               value={index.toString()}
-              overrides={radioOverrides}
+              overrides={{
```

Comment:
The option text color is different. In develop, it's `rgb(38, 39, 48)` but in this PR, it's black